data/sanskar1306.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Sanskar Kabra",
+  "type": "personal",
+  "bio": "MERN stack developer",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/sans13_k",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/sanskar-kabra-a3a347178",
+      "icon": "FaLinkedIn"
+    },
```

You probably be using an invalid icon here. It should be `FaLinkedin`. Also, remove this trailing comma `,` here.

```suggestion
      "icon": "FaLinkedin"
    }
```